uber/models.py

```diff
@@ -1144,16 +1144,14 @@ def badge_cost(self):
 
     @property
     def age_group_conf(self):
-        if self.age_group and self.age_group != c.AGE_UNKNOWN:
-            return c.AGE_GROUP_CONFIGS[self.age_group]
-        elif self.birthdate:
+        if self.birthdate:
             day = c.EPOCH.date() if date.today() <= c.EPOCH.date() else sa.localized_now().date()
             attendee_age = (day - self.birthdate).days / 365.2425
```

Comment:
oh wait duh this is the exact same line as the bug is referencing, so, ignore, we already know about it.